### PR TITLE
Made apply_effect used more consistently in regards to radiation

### PR DESCRIPTION
**Pull request by Crazylemon64**
Did a search through the codebase, thought it might be good to patch up
other things that touch the radiation variable.

### CODE REVIEW

**Comment by Markolie** [comment]
:+1: Thank you!

**Comment by Crazylemon64** [comment]
I found some cases where the game checked if the user was human, and used apply_effect if they were, and boosted .radiation if they weren't. Should I fix those up, too?

**Comment by Markolie** [comment]
Preferably, yes. You can just get rid of the human check and use apply_effect, because that works for all living mobs.

**Comment by Crazylemon64** [comment]
I'll add those too, then. A moment.

**Comment by Crazylemon64** [comment]
Healing can tweak .radiation directly, right?

**Comment by Markolie** [comment]
I think so.

**Comment by Crazylemon64** [comment]
Well, I'm done now. Merge away.

**Comment by Crazylemon64** [comment]
Though- Now that I look through, I see a few cases where the source applies 3 times as much radiation to humans. Do those matter? (Nuclear reactor, transmitter broadcasting)

**Comment by Markolie** [comment]
I don't think it matters - I don't think a lot of carbon mobs are ever around those.

File: code/modules/reagents/newchem/toxins.dm [hunk 1]
```diff
@@ -15,7 +15,7 @@ datum/reagent/polonium
 
 datum/reagent/polonium/on_mob_life(var/mob/living/M as mob)
 	if(!M) M = holder.my_atom
-	M.radiation += 8
+	M.apply_effect(8, IRRADIATE)
```
Comment (PsiOmegaDelta) [inline code/modules/reagents/newchem/toxins.dm]: Won't this basically mean that radiation suits protect against radioactive chemicals?

Comment (Markolie) [inline code/modules/reagents/newchem/toxins.dm]: Ah, you're right.

negate_armor = 1 will need to be added to the call.

Comment (Markolie) [inline code/modules/reagents/newchem/toxins.dm]: Fixed in https://github.com/ParadiseSS13/Paradise/pull/2167.

**Pull request merged**